cmake/kokkos_arch.cmake

```diff
@@ -868,7 +868,7 @@ IF (KOKKOS_ARCH_VOLTA70 OR KOKKOS_ARCH_VOLTA72)
   SET(KOKKOS_ARCH_VOLTA ON)
 ENDIF()
 
-IF (KOKKOS_ARCH_AMPERE80 OR KOKKOS_ARCH_AMPERE86 OR KOKKOS_ARCH_ADA89)
```

Comment:
That's a good question.  It likely needs to be added in the first location you mention, although I expect it would compile w/o it.
For all the others in ACC, OMP, and SYCL, we discussed yesterday adding a `KOKKOS_ARCH_NVIDIA_GPU` macro because that is really what was needed there.